guides/hosting/infrastructure/reverse-http-cache.md

```diff
@@ -273,5 +273,146 @@ if (obj.hits > 0) {
 #set resp.http.X-Cache-Hits = obj.hits;
 ```
 
+## Configure Fastly
 
+Fastly is supported since Shopware 6.4.11.0 out of the box with some configurations. To enable it we need to create a new file in `config/packages/storefront.yaml`
 
+```yaml
+storefront:
+    csrf:
+        enabled: true
+        mode: ajax
+    reverse_proxy:
+        enabled: true
+        fastly:
+          enabled: true
+          api_key: '<personal-token-from-fastly>'
+          service_id: '<service-id>'
+```
+
+### Fastly VCL Snippets
+
+Additionally we need to setup some VCL Snippets in the Fastly interface:
+
+**Name:** Normalize URLs
+
+**Subroutine:** vcl_recv
+
+```
+# Mitigate httpoxy application vulnerability, see: https://httpoxy.org/
+unset req.http.Proxy;
+
+# Strip query strings only needed by browser javascript. Customize to used tags.
+if (req.url ~ "(\?|&)(pk_campaign|piwik_campaign|pk_kwd|piwik_kwd|pk_keyword|pixelId|kwid|kw|adid|chl|dv|nk|pa|camid|adgid|cx|ie|cof|siteurl|utm_[a-z]+|_ga|gclid)=") {
+    # see rfc3986#section-2.3 "Unreserved Characters" for regex
+    set req.url = regsuball(req.url, "(pk_campaign|piwik_campaign|pk_kwd|piwik_kwd|pk_keyword|pixelId|kwid|kw|adid|chl|dv|nk|pa|camid|adgid|cx|ie|cof|siteurl|utm_[a-z]+|_ga|gclid)=[A-Za-z0-9\-\_\.\~]+&?", "");
+}
+set req.url = regsub(req.url, "(\?|\?&|&)$", "");
+
+# Normalize query arguments
+set req.url = querystring.sort(req.url);
+
+# Make sure that the client ip is forward to the client.
+if (req.http.x-forwarded-for) {
+    set req.http.X-Forwarded-For = req.http.X-Forwarded-For + ", " + client.ip;
+} else {
+    set req.http.X-Forwarded-For = client.ip;
+}
+
+# Normalize Accept-Encoding header
+# straight from the manual: https://www.varnish-cache.org/docs/3.0/tutorial/vary.html
+if (req.http.Accept-Encoding) {
+    if (req.url ~ "\.(jpg|png|gif|gz|tgz|bz2|tbz|mp3|ogg)$") {
+        # No point in compressing these
+        unset req.http.Accept-Encoding;
+    } elsif (req.http.Accept-Encoding ~ "gzip") {
+        set req.http.Accept-Encoding = "gzip";
+    } elsif (req.http.Accept-Encoding ~ "deflate") {
+        set req.http.Accept-Encoding = "deflate";
+    } else {
+        # unkown algorithm
+        unset req.http.Accept-Encoding;
+    }
+}
+
+if (req.method != "GET" &&
+    req.method != "HEAD" &&
+    req.method != "PUT" &&
+    req.method != "POST" &&
+    req.method != "TRACE" &&
+    req.method != "OPTIONS" &&
+    req.method != "PATCH" &&
+    req.method != "DELETE") {
+    /* Non-RFC2616 or CONNECT which is weird. */
+    return (pass);
+}
+
+# We only deal with GET and HEAD by default
+if (req.method != "GET" && req.method != "HEAD") {
+    return (pass);
+}
+
+# Don't cache Authenticate & Authorization
+if (req.http.Authenticate || req.http.Authorization) {
+    return (pass);
+}
+
+# Always pass these paths directly to php without caching
+# Note: virtual URLs might bypass this rule (e.g. /en/checkout)
+if (req.url ~ "^/(checkout|account|admin|api)(/.*)?$") {
```

Comment:
You seems to be evaluating URL path here, use [req.url.path](https://developer.fastly.com/reference/vcl/variables/client-request/req-url-path/) instead
```suggestion
if (req.url.path ~ "^/(checkout|account|admin|api)(/.*)?$") {
```